pkg/storaged/pages.jsx

```diff
@@ -814,16 +814,16 @@ export const StoragePage = ({ location, plot_state }) => {
     }
 
     return (
-        <Page id="storage">
+        <Page id="storage" className={client.in_anaconda_mode() ? "anaconda-mode" : ""}>
             { parent_crumbs.length > 0 &&
-            <PageBreadcrumb stickyOnBreakpoint={{ default: "top" }}>
+            <PageBreadcrumb stickyOnBreakpoint={client.in_anaconda_mode() ? {} : { default: "top" }}>
                 <Breadcrumb>
                     { parent_crumbs }
                     <BreadcrumbItem isActive>{page_display_name(page)}</BreadcrumbItem>
                 </Breadcrumb>
             </PageBreadcrumb>
             }
-            <PageSection isFilled={false}>
+            <PageSection isFilled={false} padding={client.in_anaconda_mode() ? { default: "noPadding" } : {}}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19876-20240124-111158-08af2ed3-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)